canvas_modules/common-canvas/src/common-properties/controls/checkbox/checkbox.scss

```diff
@@ -25,3 +25,11 @@
 		padding: 1px; // Allow space for checkbox borders to show when embedded within a table.
 	}
 }
+.properties-checkbox-label {
+	display: flex;
+	align-items: center;
+	.tooltip-container {
+		height: 16px;
```

Comment:
Is there a reason you need to add this?  I'm seeing the icon being a little mis-aligned yet
![Screen Shot 2020-11-16 at 8 49 25 AM](https://user-images.githubusercontent.com/21126000/99266761-bd7e5080-27e8-11eb-9f88-0fd3c925d5e5.png)